### PR TITLE
(maint) Add acceptance tests for node classification

### PR DESCRIPTION
**Pull request by ekinanp**
Signed-off-by: Enis Inan <enis.inan@puppet.com>

### CODE REVIEW

File: spec/support/acceptance/install_pe.sh [hunk 1]
```diff
@@ -38,6 +38,9 @@ if [ -z "$version" ]; then
 
   ## Create and configure Certs
   echo "autosign = true" >> /etc/puppetlabs/puppet/puppet.conf
+
+  ## Setup the RBAC token
+  echo 'pie' | puppet access login --lifetime 1y --username admin
```
Comment (ekinanp) [inline spec/support/acceptance/install_pe.sh:43]: This is needed by `puppet task run`

File: spec/acceptance/node_classification_spec.rb [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+    {
+      instance: servicenow_instance.uri,
+      user: servicenow_config['user'],
+      password: servicenow_config['password'],
+    }
+  end
+
+  test_environment = 'node_classification_tests'
+  test_environment_code_dir = "/etc/puppetlabs/code/environments/#{test_environment}"
```
Comment (RandomNoun7) [inline spec/acceptance/node_classification_spec.rb:16]: Assigning variables outside of an rspec construct that controls when the code is called can lead to thorny scope issues. Can these be turned into a `let` or something like that?

Comment (ekinanp) [reply spec/acceptance/node_classification_spec.rb:16]: No can do unfortunately. `let` is only allowed for `it`/`before(:each)` blocks (something about "per-test state"). Made them constants instead.

Comment (ekinanp) [reply spec/acceptance/node_classification_spec.rb]: Added a comment explaining this too.

**Pull request merged**